<launch>
	<rosparam file="$(find gamesh)/config/default.yaml" command="load" />
	
	<node name="gamesh_node"
		pkg="gamesh"
		type="gamesh_node"
		output="screen"
		required="true">
		
	</node>
	
</launch>
